loadtest/mock-oauth-cfn.yml

```diff
@@ -119,28 +119,23 @@ Resources:
           }
 
           const HANDLERS = {
-
-            'GET:/config': function(event, context, callback) {
```

Comment:
Should we update the comment at the top that references `GET /config`?

Reply:
yes, good catch, thanks